deepchecks/base/dataset.py

```diff
@@ -198,10 +206,9 @@ def from_numpy(
         Raises:
             DeepchecksValueError:
                 if receives zero or more than two numpy arrays;
-                if features (args[0]) is not two dimensional numpy array;
+                if columns (args[0]) is not two dimensional numpy array;
                 if labels (args[1]) is not one dimensional numpy array;
                 if features array or labels array is empty;
-                if features and labels arrays are not of the same size;
```

Comment:
Why did you remove this line? (seems like an error...)

Reply:
I just moved this validation from method `from_numpy` to `init`, so left in the docstring only direct exceptions.